### PR TITLE
Added 'glass' property to 'Cocktail' and edited ARFragment accordingly

### PR DESCRIPTION
**Pull request by shahafhermann**

### CODE REVIEW

File: app/src/main/java/com/example/cocktails/ItemDetailsActivity.kt [hunk 1]
```diff
@@ -16,7 +16,9 @@ class ItemDetailsActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.tabbed_item)
 
-        val sectionsPagerAdapter = SectionsPagerAdapter(this, supportFragmentManager)
+        val cocktail = intent.getParcelableExtra<Cocktail>("cocktail")
+
+        val sectionsPagerAdapter = SectionsPagerAdapter(this, supportFragmentManager, cocktail)
```
Comment (zoharhen) [inline app/src/main/java/com/example/cocktails/ItemDetailsActivity.kt:21]: @shahafhermann maybe change to: cocktail!! (not a nullable object)

Comment (shahafhermann) [reply app/src/main/java/com/example/cocktails/ItemDetailsActivity.kt:21]: Yeah why not. Couldn't hurt.

File: app/src/main/java/com/example/cocktails/ScrollingActivity.kt [hunk 2]
```diff
@@ -33,7 +34,9 @@ import kotlinx.android.synthetic.main.filter_dialog.view.*
 
 
 @Parcelize
-data class Cocktail(val name: String, val type: String, val steps: Array<String>, val ingredients: Array<String>, val image: String, val isCustom: Boolean = false): Parcelable
+data class Cocktail(val name: String, val type: String, val steps: Array<String>,
+                    val ingredients: Array<String>, val image: String, val isCustom: Boolean = false,
+                    val glass: Uri = Uri.parse("Water Glass.sfb")): Parcelable
```
Comment (zoharhen) [inline app/src/main/java/com/example/cocktails/ScrollingActivity.kt:39]: @shahafhermann passing a URI object triggers null pointer exception. consider passing a string instead? I just pushed a HF (hot-fix) for this issue, and added (temporary of course) this URI to the AR class.

Comment (shahafhermann) [reply app/src/main/java/com/example/cocktails/ScrollingActivity.kt:39]: I've actually tested this so it's very weird. I'll try to figure out why this happens, and if I can't make it work - than yes, we should convert this to String.

**Pull request merged**